example/web-node-sample/node-server.js

```diff
@@ -0,0 +1,111 @@
+  
+  'scalar.dl.client.private_key_pem': '-----BEGIN EC PRIVATE KEY-----\n' +
+      'MHcCAQEEICcJGMEw3dyXUGFu/5a36HqY0ynZi9gLUfKgYWMYgr/IoAoGCCqGSM49\n' +
+      'AwEHoUQDQgAEBGuhqumyh7BVNqcNKAQQipDGooUpURve2dO66pQCgjtSfu7lJV20\n' +
+      'XYWdrgo0Y3eXEhvK0lsURO9N0nrPiQWT4A==\n-----END EC PRIVATE KEY-----\n',
+  
+  'scalar.dl.client.cert_pem': '-----BEGIN CERTIFICATE-----\n' +
+      'MIICizCCAjKgAwIBAgIUMEUDTdWsQpftFkqs6bCd6U++4nEwCgYIKoZIzj0EAwIw\n' +
+      'bzELMAkGA1UEBhMCSlAxDjAMBgNVBAgTBVRva3lvMQ4wDAYDVQQHEwVUb2t5bzEf\n' +
+      'MB0GA1UEChMWU2FtcGxlIEludGVybWVkaWF0ZSBDQTEfMB0GA1UEAxMWU2FtcGxl\n' +
+      'IEludGVybWVkaWF0ZSBDQTAeFw0xODA5MTAwODA3MDBaFw0yMTA5MDkwODA3MDBa\n' +
+      'MEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIEwpTb21lLVN0YXRlMSEwHwYDVQQKExhJ\n' +
+      'bnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNC\n' +
+      'AAQEa6Gq6bKHsFU2pw0oBBCKkMaihSlRG97Z07rqlAKCO1J+7uUlXbRdhZ2uCjRj\n' +
+      'd5cSG8rSWxRE703Ses+JBZPgo4HVMIHSMA4GA1UdDwEB/wQEAwIFoDATBgNVHSUE\n' +
+      'DDAKBggrBgEFBQcDAjAMBgNVHRMBAf8EAjAAMB0GA1UdDgQWBBRDd2MS9Ndo68PJ\n' +
+      'y9K/RNY6syZW0zAfBgNVHSMEGDAWgBR+Y+v8yByDNp39G7trYrTfZ0UjJzAxBggr\n' +
+      'BgEFBQcBAQQlMCMwIQYIKwYBBQUHMAGGFWh0dHA6Ly9sb2NhbGhvc3Q6ODg4OTAq\n' +
+      'BgNVHR8EIzAhMB+gHaAbhhlodHRwOi8vbG9jYWxob3N0Ojg4ODgvY3JsMAoGCCqG\n' +
+      'SM49BAMCA0cAMEQCIC/Bo4oNU6yHFLJeme5ApxoNdyu3rWyiqWPxJmJAr9L0AiBl\n' +
+      'Gc/v+yh4dHIDhCrimajTQAYOG9n0kajULI70Gg7TNw==\n-----END CERTIFICATE-----\n',
+};
+
+const registerCertificateFunction = (async (serializedBinary) => {
+  const clientService = new ClientServiceWithBinary(properties);
+  try {
+    const r = await clientService.registerCertificate(
+        Uint8Array.from(Buffer.from(serializedBinary, 'binary')));
+    return r;
+  } catch (e) {
+    throw new Error(`${e.message}`);
+  }
+});
+
+const registerContractsFunction = (async (serializedBinary) => {
+  const clientService = new ClientServiceWithBinary(properties);
+  try {
+    const r = await clientService.registerContract(
+        Uint8Array.from(Buffer.from(serializedBinary, 'binary')));
+    return r;
+  } catch (e) {
+    throw new Error(`${e.message}`);
+  }
+});
+
+const listContractsFunction = (async (serializedBinary) => {
+  const clientService = new ClientServiceWithBinary(properties);
+  try {
+    const r = await clientService.listContracts(
+        Uint8Array.from(Buffer.from(serializedBinary, 'binary')));
+    return r;
+  } catch (e) {
+    throw new Error(`${e.message}`);
+  }
+});
+
+let result;
+app.post('/register-certificate', async (req, res) => {
+  const receivedBinaryRequest = req.body.bin;
+  try {
+    result = await registerCertificateFunction(receivedBinaryRequest);
+    res.send(result);
+  } catch (e) {
+    // CERTIFICATE_ALREADY_REGISTERED: 405
+    console.error(e.message);
+    // TODO scalardl not throwing correct error code
+    res.status(400 + Number(e.message.charAt(0))).send({error: e.message});
```

Comment:
I guess it should throw `406` instead of just `6`, that's why i add 6 there to make 406. I got it from [here](https://github.com/scalar-labs/scalardl-web-client-sdk/blob/master/README.md)

Reply:
Ah, it's fishy. 
After discussing with Plenty, this is a real bug. 
When the binary request is executed, the node client is not correctly parsing Scalar DL error which is why you got the wrong error code.

Reply:
Plenty will look into it.